components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/SelfRegistrationConfigImpl.java

```diff
@@ -144,6 +146,8 @@ public Map<String, String> getPropertyDescriptionMapping() {
                 "Length of the OTP for SMS and e-mail verifications. OTP length must be 4-10.");
         descriptionMapping.put(IdentityRecoveryConstants.ConnectorConfig.SEND_CONFIRMATION_NOTIFICATION,
                 "Enable user account confirmation when the user account is not locked on creation");
+        descriptionMapping.put(IdentityRecoveryConstants.ConnectorConfig.SHOW_USERNAME_UNAVAILABILITY,
+                "Show a descriptive error message to the user if the username is already taken. However, this leads to username enumeration");
```

Comment:
```suggestion
                "Show a descriptive error message to the user if the username is already taken. However, this may lead to username enumeration");
```